pyvista/plotting/charts.py

```diff
@@ -1374,40 +1371,36 @@ def show(
 
         Parameters
         ----------
-        off_screen : bool, optional
+        off_screen : bool, default: :attr:`pyvista.global_theme.off_screen
+            <pyvista.themes.DefaultTheme.off_screen>`
             Plots off screen when ``True``.  Helpful for saving screenshots
-            without a window popping up.  Defaults to active theme setting in
-            :attr:`pyvista.global_theme.full_screen
-            <pyvista.themes.DefaultTheme.full_screen>`.
+            without a window popping up.
```

Comment:
```suggestion
            without a window popping up. Defaults to active theme setting.
```

Not sure if this adds something so I'm also fine with removing it entirely as you did. But make sure to make it consistent with the other parameters (e.g. `full_screen` still has this comment).